### PR TITLE
Display GUI integration

### PR DESCRIPTION
**Pull request by brianhelba**

### CODE REVIEW

File: server/api/user.py [hunk 1]
```diff
@@ -205,6 +219,13 @@ def attachUserApi(user):
     events.bind('rest.post.user.after', 'onPostUser', onPostUser)
     events.bind('rest.get.user/password/temporary/:id.after', 'onGetPasswordTemporaryId',
                 onGetPasswordTemporaryId)
+
+    # TODO: Bind these in a more general way, as other places (e.g. OAuth) may also set the cookie
+    events.bind('rest.post.user.after', 'onSetCookie', onSetCookie)
+    events.bind('rest.get.user/authentication.after', 'onSetCookie', onSetCookie)
+    events.bind('rest.get.user/password/emporary/:id.after', 'onSetCookie', onSetCookie)
```
Comment (msmolens) [inline server/api/user.py]: ^ temporary

File: server/__init__.py [hunk 2]
```diff
@@ -163,6 +163,10 @@ def load(info):
         prepend=True)
 
     registerPluginWebroot(Webroot(), info['name'])
+    registerPluginWebroot(
+        Webroot(os.path.join(getPluginDir(), 'isic_archive', 'server', 'webroot_integration.mako')),
```
Comment (msmolens) [inline server/__init__.py]: Nit: Plugin dir (`.../isic_archive`) is available as `info['pluginRootDir']`.

**Comment by brianhelba** [comment]
@msmolens Fixed.

**Comment by brianhelba** [comment]
Merging now.

**Pull request merged**